mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -446,20 +458,18 @@ def _compute_mic(self, E, C, seed_idcs, target_idcs, U_bar_aa, U_bar_bb, con_i):
         alpha = V_seeds[times[:, None], freqs, :, w_seeds.argmax(axis=2)]
         beta = V_targets[times[:, None], freqs, :, w_targets.argmax(axis=2)]
 
+        # Part of Eqs. 46 & 47; i.e. transform filters to channel space
+        alpha_Ubar = np.matmul(U_bar_aa, np.expand_dims(alpha, axis=3))
+        beta_Ubar = np.matmul(U_bar_bb, np.expand_dims(beta, axis=3))
```

Comment:
Do we use `np.matmul` here instead of the `@` operator because of specific broadcasting rules?

Reply:
This is actually something I wanted to get thoughts on.

Back when I first added these methods I used `np.matmul` instead of `@` because I didn't see `@` used anywhere else in MNE-Connectivity, so I wasn't sure if the explicit `np.matmul` was preferred. But I since realised that `@` is used all over the place in the main MNE package.

I feel `np.matmul` hinders readability somewhat so I wanted to open a PR and switch to `@`, but it just hasn't been a priority for me. Perhaps after v0.7 is out I can do this.

Reply:
`@` is preferred. There may be some old instanced of `np.matmul` left. Feel free to change these if it doesn't break anything. I read somewhere that the broadcasting rules are a little different. In this case, we explicitly call `expand_dims` so I don't think broadcasting is going to be an issue.

Reply:
Okay I can make this switch. Thankfully we have the regression tests which should pick up if this messes anything up.

Reply:
looks like these and a few other places still need to be switched to `@`

Reply:
Made the switch in a separate branch to make reviewing easier but didn't submit a PR for (didn't want to confuse things for the 0.7 release).

Have submitted now in #201. Regression tests pass, so no broadcasting silliness introduced.

Reply:
Merged now